README.md

```diff
@@ -15,254 +16,183 @@
+## **Built With**
+Celo Composer is built on the Celo ecosystem and supports multiple frameworks and libraries:
 
-```text
-Do you want to use a template?
-```
+- [Celo](https://celo.org/)
+- [Solidity](https://docs.soliditylang.org/)
+- [Hardhat](https://hardhat.org/)
+- [React.js](https://reactjs.org/)
+- [Next.js](https://nextjs.org/)
+- [Viem](https://viem.sh/)
+- [Tailwind CSS](https://tailwindcss.com/)
 
-- Step 5: Select a Template: If you chose to use a template, you will be prompted to select a template from the list provided.
+---
 
-```text
-- Minipay
-- Valora
-```
+## **Prerequisites**
+Ensure you have the following installed:
 
-- Step 6: Provide the Project Owner's Name: You will be asked to enter the project owner's name.
+- **Node.js** (v20 or higher)
+- **Git** (v2.38 or higher)
 
-```text
-Project Owner name:
-```
+---
 
-- Step 7: Wait for Project Creation: The CLI will now create the project based on your inputs. This may take a few minutes.
+## **How to use Celo Composer**
 
-- Step 8: Follow the instructions to start the project. The same will be displayed on the console after the project is created.
+The easiest way to start is by using the `@celo/celo-composer` CLI tool. It helps bootstrap dApps with different templates.
 
-```text
-🚀 Your starter project has been successfully created!
+```bash
+npx @celo/celo-composer@latest create
 ```
 
-## Install Dependencies
-
+### **Installation Steps**
+1. **Enter a project name**
+2. **Choose whether to use Hardhat** (Yes/No)
+3. **Select a template** (Minipay, Valora, etc.)
+4. **Provide project owner details**
+5. **Wait for project creation to complete**
+6. **Follow the on-screen instructions to start your project**
 
-Once your custom dApp has been created, just install dependencies, either with yarn:
+---
 
-```bash
-   yarn
-```
-
-If you prefer npm, you can run:
+## **Install Dependencies**
+Once the project is set up, install dependencies:
 
 ```bash
-   npm install
+yarn
+# or
+npm install
 ```
 
-## Deploy a Smart Contract
-
-Find the detailed instructions on how to run your smart contract in [packages/hardhat/README.md](./packages/hardhat/README.md).
+---
 
-For quick development follow these three steps:
+## **Deploy a Smart Contract**
+For detailed instructions, refer to [`packages/hardhat/README.md`](./packages/hardhat/README.md).
 
-1. Change `packages/hardhat/env.template` to `packages/hardhat/env` and add your `PRIVATE_KEY` into the `.env` file.
-2. Make sure your wallet is funded when deploying to testnet or mainnet. You can get test tokens for deploying it on Alfajores from the [Celo Faucet](https://faucet.celo.org/alfajores).
-3. Run the following commands from the `packages/hardhat` folder to deploy your smart contract to the Celo Testnet Alfajores:
+Quick steps:
+1. Rename `packages/hardhat/.env.template` to `packages/hardhat/.env` and add your `PRIVATE_KEY`.
+2. Ensure your wallet has test funds from the [Celo Faucet](https://faucet.celo.org/alfajores).
```

Comment:
This sentence might not be clear to someone who is just starting in Web3

2. Ensure your wallet is funded before deploying a contract. You can get test tokens from the [Celo Faucet](https://faucet.celo.org/alfajores).